src/client/app/containers/ExportContainer.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { connect } from 'react-redux';
+import ExportComponent from '../components/ExportComponent';
+
+/**
+ * @param {State} state
+ * @return {{meterInfo: *, selectedMeters: Array}}
+ */
+function mapStateToProps(state) {
+	const timeInterval = state.graph.timeInterval;
+	const data = { datasets: [] };
+	let readingsData;
+	const chart = state.graph.chartToRender;
+	const barDuration = state.graph.barDuration;
+
+	for (const meterID of state.graph.selectedMeters) {
+		if (chart === 'line') {
```

Comment:
In reducers/graph.js, there is `chartTypes`. Even though it doesn't really do anything, you should use them instead of the string 'line' and 'bar' so that we don't get confused when we start adding more types of charts.